pkg/reconciler/ingress/ingress_test.go

```diff
@@ -1115,7 +1094,12 @@ func patchAddFinalizerAction(ingressName, finalizer string) clientgotesting.Patc
 	action := clientgotesting.PatchActionImpl{
 		Name: ingressName,
 	}
-	patch := fmt.Sprintf(`{"metadata":{"finalizers":[%q],"resourceVersion":"v1"}}`, finalizer)
+	var patch string
+	if finalizer != "" {
+		patch = fmt.Sprintf(`{"metadata":{"finalizers":[%q],"resourceVersion":"v1"}}`, finalizer)
+	} else {
+		patch = `{"metadata":{"finalizers":[],"resourceVersion":"v1"}}`
```

Comment:
The function name is `patchAdd`, so I think it should be a separate function (or we should rename the function)